### DISCUSSION TITLE
Helper Functions Page Missing

### DISCUSSION THREAD

**Discussion by stevebarakat** [johnlindquist/kit#460] [Errors] [answered]
![Screen Shot 2021-09-29 at 8 49 43 AM](https://user-images.githubusercontent.com/3830326/135271724-76e6c422-424e-48be-bf27-ea555c0084e7.png)

**Comment by johnlindquist** [comment, marked as answer]
https://github.com/johnlindquist/kit/blob/main/src/types/kit.d.ts

and 

https://github.com/johnlindquist/kit/blob/main/src/types/lib.d.ts

**Reply by stevebarakat** [reply]
![typo](https://user-images.githubusercontent.com/3830326/135288415-f2e82bdb-a995-41db-865c-cf83756b5985.png)

**Reply by johnlindquist** [reply]
Just fyi, I'm organizing and cleaning up these definition files this week

**Reply by stevebarakat** [reply]
Oh sorry. Don't mean to be annoying. I'm done with the script so I'll be out of your hair. Thanks for all the help!

**Reply by johnlindquist** [reply]
Happy to help. I wasn't annoyed, just giving a heads-up that some stuff is moving around. Mostly for the new TypeScript script support.
